src/Apps/Order/Routes/Shipping2/index.tsx

```diff
@@ -158,14 +135,10 @@ export const ShippingRoute: FC<ShippingProps> = props => {
         error.code === "missing_country" ||
         error.code === "missing_postal_code"
       ) {
-        trackEvent({
-          action: ActionType.errorMessageViewed,
-          context_owner_type: OwnerType.ordersShipping,
-          context_owner_id: props.order.internalID,
-          title: "Invalid address",
-          message:
-            "There was an error processing your address. Please review and try again.",
+        errorMessageViewed({
           error_code: error.code,
+          title: "Missing information",
+          message: "Please complete all required fields.",
```

Comment:
**issue:** This title and message don't match what's displayed to the user in the error dialog.